### ISSUE TITLE
Bug: Marker cluster image does not appear (GmapCluster)

### ISSUE THREAD

**Issue by danbohea** [diegoazh/gmap-vue#150]
**Describe the bug**
When using marker clustering the image that replaces the clustered markers is invisible. The markers are hidden according to the `minimumClusterSize` prop as expected but they are replaced with nothing.

**To Reproduce**
1. Import the component: `import GmapCluster from 'gmap-vue/dist/components/cluster'`
2. Register the component: `components: { GmapCluster },`
3. Wrap the `gmap-marker` element with the `gmap-cluster` element: `<gmap-cluster>[gmap markers...]</gmap-cluster>`
4. Ensure that you have some markers that are positioned reasonably close together to trigger clustering at certain zoom levels.

**Expected behavior**
When zoomed out, closely grouped markers should be replaced with a cluster icon that denotes how many markers it's replaced. When zooming closer in, the markers should return and the cluster icon should be hidden.

**Current behavior**
Markers are replaced at the expected zoom level but they are replaced with seemingly nothing (no icon, no text).

**Screenshots**

Zoomed in:
![image](https://user-images.githubusercontent.com/6885065/110937498-5bfb6800-832a-11eb-8fa8-4dbaf2cd0950.png)

Zoomed out:
![image](https://user-images.githubusercontent.com/6885065/110937555-6f0e3800-832a-11eb-83a7-ce57a2464ed3.png)

 - OS: Manjaro Linux
 - Browser: Firefox
 - Version: 86

**Additional context**
The marker clustering example in the docs doesn't work either but this may be a different issue (the clustering never occurs, rather than the cluster icon being invisible). https://diegoazh.github.io/gmap-vue/examples/basic-marker-cluster.html#live-example

## Versions

- Node: 14.16.0
- NPM: 6.14.11

## Package manager

NPM

## Plugin version

1.5.0

**Comment by danbohea** [comment]
Instantly closed? Really? I followed your issue template to the letter. You really need to consider tweaking your issue bots.

**Comment by danbohea** [comment]
@diegoazh I followed your issue template but got insta-closed. Can you please either re-open this issue or tell me what I've got to do to submit an issue that stays open for more than 1 second?

**Comment by danbohea** [comment]
It turns out that adding [this folder](https://github.com/eregnier/vue2-gmap-custom-marker-demo/tree/master/public/images) fixes the issue for now. Thanks to the author of [this plugin](https://github.com/eregnier/vue2-gmap-custom-marker). Please don't consider this resolved though. This still needs fixing in this project.

**Comment by diegoazh** [comment]
> Instantly closed? Really? I followed your issue template to the letter. You really need to consider tweaking your issue bots.

why you removed the Desktop and Smartphone sections?

**Comment by danbohea** [comment]
> if you don't have information for those sections leave them empty but you don't need to remove the sections
OK, thanks for the info. You might want to make that clearer in the template instructions. Either way, your bot is waaaaay too over the top.

**Comment by CHOMNANP** [comment]
I'm having the same issue. Not sure how to fix this.

**Comment by bkostjens** [comment]
Same issue, I just switched from vue2-google-maps to gmap-vue. vue2-google-maps does not have this issue.

**Comment by danbohea** [comment]
Honestly I've found everything easier by just using the Google Maps API directly. The support is better, it's more powerful, you have less dependencies etc.

**Comment by clcoco** [comment]
> It turns out that adding [this folder](https://github.com/eregnier/vue2-gmap-custom-marker-demo/tree/master/public/images) fixes the issue for now. Thanks to the author of [this plugin](https://github.com/eregnier/vue2-gmap-custom-marker). Please don't consider this resolved though. This still needs fixing in this project.

Where did you add it? I've got the same problem

**Comment by danbohea** [comment]
@clcoco I think it was in public/images/ - not sure though, I'm not using this library any more as I've just had too many issues. Use the Google Maps API directly, you'll have a way smoother ride once you're up & running. Start here: https://markus.oberlehner.net/blog/using-the-google-maps-api-with-vue/

**Comment by clcoco** [comment]
Unfortunately I'm a bit on a hurry, but it doesn't seem to be that hard so I'll give it a shot next. 
Anyway, I updated the previous comment with the solution

**Comment by dirk-ecker** [comment]
> @bkostjens @CHOMNANP @danbohea and @dirk-ecker the cluster component was fixed on v2.0.3, please can you test it again. Regards.

Thank you but I can't get it working. He always complains

```
This dependency was not found:

* MarkerClusterer in ./node_modules/gmap-vue/dist/gmap-vue.js
```

and I have checked this

```
"dependencies": {
    "@googlemaps/markerclusterer": "^1.0.23",

```

and also that 
`"marker-clusterer-plus": "^2.1.4"`

in my package.json.

My view uses

```
import GmapMap from 'gmap-vue/dist/gmap-vue'
import GmapCluster from 'gmap-vue/dist/components/cluster-icon'
import GmapMarker from 'gmap-vue/dist/components/marker-icon'
```

Can you help?

**Comment by diegoazh** [comment]
> > @bkostjens @CHOMNANP @danbohea and @dirk-ecker the cluster component was fixed on v2.0.3, please can you test it again. Regards.
> 
> Thank you but I can't get it working. He always complains
> 
> ```
> This dependency was not found:
> 
> * MarkerClusterer in ./node_modules/gmap-vue/dist/gmap-vue.js
> ```
> 
> and I have checked this
> 
> ```
> "dependencies": {
>     "@googlemaps/markerclusterer": "^1.0.23",
> ```
> 
> and also that `"marker-clusterer-plus": "^2.1.4"`
> 
> in my package.json.
> 
> My view uses
> 
> ```
> import GmapMap from 'gmap-vue/dist/gmap-vue'
> import GmapCluster from 'gmap-vue/dist/components/cluster-icon'
> import GmapMarker from 'gmap-vue/dist/components/marker-icon'
> ```
> 
> Can you help?

Thank you @dirk-ecker you remember me that I need to update the documentation, you need to install `@googlemaps/markerclusterer`, the market cluster component was renamed to `Cluster` and it is exported from the library in a `components` object you need to do something like the following

```vue
<template>
  <div>
    <gmap-map :center="center" :zoom="7" style="width: 100%; height: 500px">
      <gmap-cluster>
        <gmap-marker
          v-for="(m, index) in markers"
          :position="m.position"
          :clickable="true"
          :draggable="true"
          @click="center = m.position"
          :key="index"
        ></gmap-marker>
      </gmap-cluster>
    </gmap-map>
  </div>
</template>

<script>
import { components } from "gmap-vue";

export default {
  ....
  components: {
    "gmap-cluster": components.Cluster
  }
};
</script>
```

**Comment by danbohea** [comment]
Please stop asking me to test your library. As I already mentioned above, I'm no longer using it.

**Comment by diegoazh** [comment]
> dirk-ecker

No problem, sorry for that.

**Comment by dirk-ecker** [comment]
> > > @bkostjens @CHOMNANP @danbohea and @dirk-ecker the cluster component was fixed on v2.0.3, please can you test it again. Regards.
> > 
> > 
> > Thank you but I can't get it working. He always complains
> > ```
> > This dependency was not found:
> > 
> > * MarkerClusterer in ./node_modules/gmap-vue/dist/gmap-vue.js
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > and I have checked this
> > ```
> > "dependencies": {
> >     "@googlemaps/markerclusterer": "^1.0.23",
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > and also that `"marker-clusterer-plus": "^2.1.4"`
> > in my package.json.
> > My view uses
> > ```
> > import GmapMap from 'gmap-vue/dist/gmap-vue'
> > import GmapCluster from 'gmap-vue/dist/components/cluster-icon'
> > import GmapMarker from 'gmap-vue/dist/components/marker-icon'
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Can you help?
> 
> Thank you @dirk-ecker you remember me that I need to update the documentation, you need to install `@googlemaps/markerclusterer`, the market cluster component was renamed to `Cluster` and it is exported from the library in a `components` object you need to do something like the following
> 
> ```
> <template>
>   <div>
>     <gmap-map :center="center" :zoom="7" style="width: 100%; height: 500px">
>       <gmap-cluster>
>         <gmap-marker
>           v-for="(m, index) in markers"
>           :position="m.position"
>           :clickable="true"
>           :draggable="true"
>           @click="center = m.position"
>           :key="index"
>         ></gmap-marker>
>       </gmap-cluster>
>     </gmap-map>
>   </div>
> </template>
> 
> <script>
> import { components } from "gmap-vue";
> 
> export default {
>   ...
>   components: {
>     "gmap-cluster": components.Cluster
>   }
> };
> </script>
> ```

Okay, in the end I got it running.

**Comment by diegoazh** [comment]
Excellent I'll mark this issue as solved.
